stestr/commands/init.py

```diff
@@ -38,8 +40,11 @@ def init(repo_type='file', repo_url=None):
         for failures.
     :rtype: int
     """
-
-    util.get_repo_initialise(repo_type, repo_url)
+    try:
+        util.get_repo_initialise(repo_type, repo_url)
+    except FileExistsError:
+        stdout.write('The specified repository already exists\n')
+        return 0
```

Comment:
I'm not sure about this return code. I kinda want to make this a failure condition, because we can't actually be sure if it's a stestr file repository or not from this. Just that the directory exists. On the sql side it's an idempotent function, but that's because it has a concrete api endpoint and the migration scripts it runs are idempotent. But because it's idempotent on sql I thought I should return 0 here, but there isn't actually a guarantee the repository exists in this case. 

Does anyone have any thoughts on this? I'm strongly leaning towards changing this to be a failure condition before we merge this.